hw/top_earlgrey/data/ip/chip_rstmgr_testplan.hjson

```diff
@@ -82,7 +82,9 @@
         "RSTMGR.RESET_INFO.CLEAR",
       ]
       tests: ["chip_rv_dm_ndm_reset_req"]
-      bazel: []
+      bazel: [
+        "//sw/device/tests:ndm_reset_req",
```

Comment:
Is this correct? I see your `opentitan_test`s are called `rv_dm_ndm_reset_req_$lc_state`.